commands/download.js

```diff
@@ -0,0 +1,42 @@
+const axios = require('axios');
+const chalk = require('chalk')
+var fs = require('fs');
+
+async function download(id) {
+    console.log(chalk.yellow(`Downloading trace ${id}...`))
+    let url = `https://api.gotrace.world/v1/public/trace/${id}`
+    console.log(chalk.green.bold('The public trace ' + id + ' is being downloaded'))
+    try {
+        response = await axios.get(url)
```

Comment:
Why wouldn't you use fetch instead of axios?

Reply:
@treeder replaced axios with fetch (no difference)

Reply:
Well, the difference is there's no dependencies with fetch.